corehq/apps/repeaters/dbaccessors.py

```diff
@@ -70,23 +70,23 @@ def get_paged_repeat_records(domain, skip, limit, repeater_id=None, state=None):
         'reduce': False,
         'limit': limit,
         'skip': skip,
+        'descending': True,
     }
 
     if repeater_id and not state:
-        kwargs['startkey'] = [domain, repeater_id]
-        kwargs['endkey'] = [domain, repeater_id, {}]
+        kwargs['endkey'] = [domain, repeater_id]
+        kwargs['startkey'] = [domain, repeater_id, {}]
```

Comment:
I'm not understanding why `startkey` and `endkey` got swapped here (with empty dict in `startkey`), but not on line 36 (where `endkey` has the empty dict). I also have never quite understood the purpose of the empty dict, so maybe if I understood that it would make more sense.